test/Certificate.t.sol

```diff
@@ -77,7 +77,7 @@ contract Certificate_burn is UpgradeableCertificate {
       address(0),
       _removalIds2,
       _removalAmounts,
-      abi.encode(address(this), 1 ether) // todo use diff. named account
+      abi.encode(address(this), 1 ether, address(0), 100) // todo use diff. named account
```

Comment:
just updating existing tests with an arbitrary filler value for these new pieces of data -- this is not an integration test